Makefile

```diff
@@ -492,5 +489,16 @@ comp-messages: .env
 messages: .env
 	@docker-compose exec web python manage.py makemessages -a
 
-lint: .env
+lint: .env ensure-mysql
 	@docker-compose run --rm web pylint impact
+
+ensure-mysql: kill-exited-containers
```

Comment:
Seven of the targets naming `ensure-mysql` as a dependency also name `kill-exited-containers` as a dependency. 

This is a little bikesheddy I admit, but in the interest of reducing verbosity and repetition could we add a target for use by the two that don't (`load-remote-db` and `lint`), like `ensure-mysql-but-keep-containers`, so we can drop it from the other seven?

Reply:
good eye!
- the `ensure-mysql` on `load-remote-db` is actually redundant since the target it executes i.e. `load-db` also executes `ensure-mysql`
- `lint` actually also needs to `kill-exited-containers` just to be sure it doesn't restart 'bad' containers.

but thanks for this, removed the `kill-exited-containers` redundancy